onnxruntime/core/providers/cpu/ml/label_encoder.h

```diff
@@ -98,7 +98,7 @@ class LabelEncoder_2 final : public OpKernel {
   // A collection of key-value pairs. Each (a_key, a_value) pair
   // means that the "a_key" in the input would be mapped to "a_value".
   // If _map doesn't contain "a_key", we use _default_value as its output.
-  std::unordered_map<TKey, TValue> _map;
+  absl::flat_hash_map<TKey, TValue> _map;
```

Comment:
Would you please use `InlinedHashMap` instead?
https://github.com/microsoft/onnxruntime/blob/2e2e6aeff646178d5606f85c33ff4c1189dbd0ef/include/onnxruntime/core/common/inlined_containers.h#L63-L76

Reply:
```suggestion
  InlinedHashMap<TKey, TValue> _map;
```

Reply:
Thanks for pointing out the header file.